change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
```suggestion
  "type": "none",
```

? 🤔

Reply:
yup, accidentally hit the default value for type